e2e/portalicious/pages/BasePage.ts

```diff
@@ -104,6 +104,12 @@ class BasePage {
     const fileChooser = await fileChooserPromise;
     await fileChooser.setFiles(filePath);
   }
+
+  async goto(path: string) {
+    const defaulLanguage = 'en-GB';
```

Comment:
Should we get this from somewhere? We cannot access NG_DEFAULT_LOCALE in playwright at this moment

Reply:
```suggestion
    const defaultLanguage = 'en-GB';
```

Besides the typo, I think this is fine. We have it hardcoded in plenty of places in the E2E tests already - and if it breaks we will hear about it in CI.